docs/web3.eth.rst

```diff
@@ -39,6 +39,11 @@ you can find the latest block number in these two ways:
         Traceback # ... etc ...
         TypeError: This data is immutable -- create a copy instead of modifying
 
+This feature is available via the ``attrdict_middleware`` which is a default middleware.
+Of note, accessing a property via attribute will break type hinting. If typing is
```

Comment:
This might be better as a `..note ::` box thing. Feel free to take or leave though.